spec/lib/miq_expression/subst_mixin_spec.rb

```diff
@@ -0,0 +1,12 @@
+RSpec.describe MiqExpression::SubstMixin do
+  let(:test_class) { Class.new { include MiqExpression::SubstMixin } }
+  let(:test_obj) { test_class.new }
+
+  describe "#exp_replace_qs_tokens" do
+    it "removes :token key from passed expression" do
+      exp = {"=" => {"field" => "Vm-active", "value" => "true"}, :token => 1}
+      test_obj.exp_replace_qs_tokens(exp, {})
+      expect(exp).to eq("=" => {"field" => "Vm-active", "value" => "true"})
```

Comment:
This looks good. But it may be good to test with a more complex expression. Something like -
```ruby
{"and"=>[{"="=>{"field"=>"Vm-active", "value"=>"true"}, :token => 1}, {"="=>{"field"=>"Vm-archived", "value"=>"true"}, :token => 1}], :token => 1}
```
Just a suggestion

Reply:
@gtanzillo ok, make sense, will do with this expression: 
`{"and"=>[{"="=>{"field"=>"Vm-active", "value"=>"true"}, :token => 1}, {"="=>{"field"=>"Vm-archived", "value"=>"true"}, :token => 1}]}`. (there is no tokens added on the level of `and` or `or`)